packages/terra-form-textarea/package.json

```diff
@@ -39,6 +41,7 @@
     "lint:scss": "stylelint src/**/*.scss",
     "test": "npm run test:jest && npm run test:nightwatch",
     "test:jest": "jest ./tests/jest/* --config ../../jestconfig.json",
-    "test:nightwatch": "nightwatch -c ../../nightwatch.conf.js"
+    "test:nightwatch": "nightwatch -c ../../nightwatch.conf.js",
```

Comment:
Can we drop the nightwatch tests in this pr?

Reply:
Not here. We still have nightwatch tests in the package